tenants/composer-ai/apps/base/app-of-apps.yaml

```diff
@@ -15,6 +15,8 @@ spec:
     targetRevision: main
     helm:
       parameters:
+        - name: clusterDomain
+          value: <REPLACE_ME>
```

Comment:
So in the catalog item version I'm expecting that the demo team makes this available as an env var to the ArgoCD instance and this is interpreted at the time that argo applies the resources.

I think that we may want to try and follow a similar pattern with the env var to avoid having to make this update every time you deploy a new cluster and make a commit.

I haven't seen this in action though so I'm not 100% sure how it will work.

Reply:
Yeah my assumption is it is using this:
https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/vars/

I can update this to work with `vars` but I wanted to wait to see exactly how it worked first and just get it up and running for now.